### PR TITLE
Code review section tidy up

### PR DESCRIPTION
**Pull request by junglebarry**
Two tidy-up suggestions:

1. Use Github labels rather than title patterns to indicate 30% or 90% review requests.  Labels are searchable in the Github interface; Labels should also lead to a neater commit log, since we won't have lots of commits prefixed "90%" being merged (or responsibility for cleaning up)

1. remove the "0%" section as that is the default state of all pull-requests, so there's no need to draw attention to it.  Tried to keep the motivation, just apply to the default state.

### CODE REVIEW

File: topics/code-reviews.md [hunk 1]
```diff
@@ -4,45 +4,39 @@ title: Code Reviews
...
-which allows developers to get early feedback on designs before dotting the i's and crossing the t's. Simply mark your
-pull request with 30% in the title to get this kind of feedback.
+which allows developers to get early feedback on approach before dotting the i's and crossing the t's. Simply label your
+pull request with a "30%" tag to get this kind of feedback.
 
-Later, when you are closer to merge, and need a pedant, re-title your pull with 90% to get feedback on the details.
+Later, when you are closer to merge, and need a pedant, re-label your pull with a "90%" tag to get feedback on the details.
 
-You'll find that smaller changes (one liners or very small features) will skip the 30% stage and go right to demanding
+You'll find that smaller changes (one liners or very small features) will skip the 30% stage and go right to requesting
```
Comment (junglebarry) [inline topics/code-reviews.md]: Language seemed quite aggressive.

File: topics/code-reviews.md [hunk 2]
```diff
@@ -4,45 +4,39 @@ title: Code Reviews
...
 90% feedback.
 
-### What is a 0% PR?
-
-These are pull requests raised by developers wishing to track their own changes, and organising their thoughts for
-later review.
+If there is neither a "30%" nor "90%" tag, please do not comment on these PRs unless the owner asks you to.
 
-Please do not comment on these PRs unless the owner asks you to.
+We used to recommend adding the 30%/90% to the pull-request title, but that makes for ugly commit logs unless youthey are cleaned up. Github labels are a more elegant solution.
```
Comment (camillef) [inline topics/code-reviews.md]:
```suggestion
We used to recommend adding the 30%/90% to the pull-request title, but that makes for ugly commit logs unless they get cleaned up before merging. Github labels are a more elegant solution.
```

File: topics/code-reviews.md [hunk 3]
```diff
@@ -4,45 +4,39 @@ title: Code Reviews
...
 
 Don't always rely on the diff - depending on the size of the changes in the pull request, it might be a good idea to
 check out the branch locally. This gives you a number of benefits:
 
-* You can open up your IDE and see the code as it was written. A variable may have been renamed - you can search in
-your IDE for the old values.
-* You can spin up the app and take it for a test drive. The code might look ok, but the UI may look terrible and
-spam the console with errors, APIs might be hard to use, and it might run like a dog.
-* You can run the local build (including tests) and see any potential breakages.
+- You can open up your IDE and see the code as it was written. A variable may have been renamed - you can search in
```
Comment (junglebarry) [inline topics/code-reviews.md]: This wasn't me, honest - prettier.  Will remove.

File: topics/code-reviews.md [hunk 4]
```diff
@@ -4,45 +4,39 @@ title: Code Reviews
...
 
-You'll find that smaller changes (one liners or very small features) will skip the 30% stage and go right to demanding
+You'll find that smaller changes (one liners or very small features) will skip the 30% stage and go right to requesting
 90% feedback.
 
-### What is a 0% PR?
-
-These are pull requests raised by developers wishing to track their own changes, and organising their thoughts for
-later review.
+If there is neither a "30%" nor "90%" tag, please do not comment on these PRs unless the owner asks you to.
```
Comment (junglebarry) [inline topics/code-reviews.md]: The 0% section is redundant - every PR without an explicit request is in this state.  I've left the statement about the default state, but I'm not even sure that is warranted.

File: topics/code-reviews.md [hunk 5]
```diff
@@ -4,45 +4,39 @@ title: Code Reviews
...
 
-Please do not comment on these PRs unless the owner asks you to.
+(We previously recommended adding the 30%/90% to the pull-request title, but that makes for ugly commit logs unless they are cleaned up. Github labels are a more elegant solution.)
```
Comment (junglebarry) [inline topics/code-reviews.md]: This is here for motivation, but that could equally well be removed and handled by this PR being a record of *why* I'm suggesting this change.

File: topics/code-reviews.md (hunk 4 above)
Comment (camillef) [inline topics/code-reviews.md]:
```suggestion
If there is neither a "30%" nor "90%" tag, please do not comment on the pull request unless the owner asks you to. Pull request are often used by developers wishing to track their own changes and organise thoughts before being presented for review. It's good practise to review your own PR before requesting a review from others.
```

File: topics/code-reviews.md (hunk 5 above)
Comment (camillef) [reply topics/code-reviews.md]: I think this is redundant personally

File: topics/code-reviews.md (hunk 4 above)
Comment (camillef) [reply topics/code-reviews.md]: Suggesting a bit more background about what the motivation of opening a PR before 30% is

File: topics/code-reviews.md [hunk 6]
```diff
@@ -4,45 +4,39 @@ title: Code Reviews
...
-
 All code at Talis must go through peer review before it can be merged into `master`. This can be done by
 raising a pull request on the relevant GitHub repository and asking your colleagues for input.
 
 ### 30% vs 90%
 
 Early feedback on an outline plan for your code is great. We follow the 30% feedback rule[^1]
-which allows developers to get early feedback on designs before dotting the i's and crossing the t's. Simply mark your
-pull request with 30% in the title to get this kind of feedback.
+which allows developers to get early feedback on approach before dotting the i's and crossing the t's. Simply label your
```
Comment (markwallsgrove) [inline topics/code-reviews.md]:
```suggestion
which allows developers to get early feedback on the approach before dotting the i's and crossing the t's. Simply label your
```

File: topics/code-reviews.md (hunk 4 above)
Comment (camillef) [reply topics/code-reviews.md]: (Also, would prefer not to use the abbreviation PR unless we define it)

Comment (markwallsgrove) [inline topics/code-reviews.md]:
```suggestion
If there is neither a "30%" nor "90%" tag, please do not comment on these PRs unless the author asks you to.
```

Comment (junglebarry) [reply topics/code-reviews.md]: I've done this definition closer to the top.

File: topics/code-reviews.md (hunk 5 above)
Comment (junglebarry) [reply topics/code-reviews.md]: Shall I just remove (I already added your feedback)?

File: topics/code-reviews.md (hunk 3 above)
Comment (junglebarry) [reply topics/code-reviews.md]: actually, it kinda fixes a broken window with spacing, so I'll leave in place unless there are objections.

File: topics/code-reviews.md (hunk 4 above)
Comment (junglebarry) [reply topics/code-reviews.md]: I think I've updated with something like the original motivation now.

Comment (junglebarry) [reply topics/code-reviews.md]: ... and added back the motivation

**Pull request merged**